manifests/profile/python.pp

```diff
@@ -16,7 +16,14 @@
 #  include st2::profile::python
 #
 class st2::profile::python {
-  if !defined(Class['::python']) {
+  if $::osfamily == "RedHat" {
+    package {'python27':
```

Comment:
Honestly I haven't dug in to the python module to see how it works.  I needed to be sure it would pull down the python2.7 package from the IUS repo.  I'll verify whether the python module makes use of yum or not.